Cohort2/team4/server/src/models/Event.ts

```diff
@@ -1,7 +1,7 @@
 import { Table, Column, PrimaryKey, Model, AutoIncrement, DataType, AllowNull } from 'sequelize-typescript';
 
-@Table({ tableName: 'event' })
-class Event extends Model {
+@Table({ tableName: 'events' })
```

Comment:
Can you add `timestamps: false` after the tableName? Just so that we don't have the timestamps generated by sequelize.